### ISSUE TITLE
How do I avoid ticktick automatically detecting new versions on the network

### ISSUE THREAD

**Issue by HiramYang** [yazdipour/unlimited-ticktick-windows#21]
How do I avoid ticktick automatically detecting new versions on the Internet? What files should I modify with dnSpy?

**Comment by SharadenduDwivedi** [comment]
the update isn't compulsary, is it?

I am using a slitly older version than the latest and I choose not to upgrade because the newest one is having trouble for me.

**Comment by HiramYang** [comment]
Yes, but I want to know what the code is for the software to detect updates, and I want the old version of the software to think it's the latest version.
